examples/doge-example/project.json

```diff
@@ -4,26 +4,14 @@
   "projectType": "application",
```

Comment:
Took `project.json` from another examples.

Reply:
I thought since it's generated it shouldn't be changed much, tho I see there are some things that I changed in the code and didn't reflect in this file